js/data_v4.js

```diff
@@ -5,6 +5,8 @@ var THIRTY_DAYS_IN_SECONDS = 2592000;
 var PAINT_TIME_THRESHOLD = 300000;
 var CONFIG_URL = 'js/config.json?';
 var prefs = null;
+// The following is the list of ping types we will display in the page.
```

Comment:
This doesn't sound correct.
These are the ping types we will load and accumulate data from - we do show more pings in the raw data etc.